src/components/ContributorSearchField.tsx

```diff
@@ -27,13 +27,15 @@ interface ContributorSearchFieldProps {
   setSelectedPerson: (val: CristinPerson | undefined) => void;
   searchTerm: string;
   setSearchTerm: (val: string) => void;
+  singleSelect?: boolean;
 }
 
 export const ContributorSearchField = ({
   selectedPerson,
   setSelectedPerson,
   searchTerm,
   setSearchTerm,
+  singleSelect = false,
```

Comment:
Kan ikke se at denne sette til noe annet enn `false` nå 🤔 
Tenkte planen var at den skulle være `true` når man legger til prosjektleder?
![bilde](https://github.com/user-attachments/assets/95283c9b-2f6c-4612-99b3-7c3f34b70526)

Reply:
Ja stemmer! Den er ikke i bruk enda, men du kan jo bruke den i koden for å se den i aksjon.

Reply:
Men rent praktisk har den allerede oppfylt kravet om å bruke checkboxer istedet for runder bokser ved multiple select da.